cmd/clusterctl/cmd/init.go

```diff
@@ -86,6 +87,7 @@ func init() {
 	initCmd.Flags().StringVarP(&io.coreProvider, "core", "", "", "Infrastructure providers to add to the management cluster. By default (empty), the cluster-api core provider is installed on the first init")
 	initCmd.Flags().StringSliceVarP(&io.infrastructureProviders, "infrastructure", "i", nil, "Infrastructure providers to add to the management cluster")
 	initCmd.Flags().StringSliceVarP(&io.bootstrapProviders, "bootstrap", "b", nil, "Bootstrap providers to add to the management cluster. By default (empty), the kubeadm bootstrap provider is installed on the first init")
+	initCmd.Flags().StringSliceVarP(&io.controlPlaneProviders, "controlplane", "c", nil, "ControlPlane providers to add to the management cluster. By default (empty), the kubeadm control plane provider is installed on the first init")
```

Comment:
nit: `-c` or `-b` is not very intuitive, I would force long-form only

Reply:
I agree that the shorthands aren't great, but given that `infrastructure` and `bootstrap` already have the shorthands, would it not be better to leave this short hand in and be consistent with the existing flags assuming they're already in use by users?

Reply:
Let's open an issue to discuss the possibility of removing the shorthands.

@JoelSpeed this version of clusterctl is 100% new in the master branch for v1alpha3, and therefore the UX is completely open for debate up until we cut the next minor release.

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/2118